src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.js

```diff
@@ -1,4 +1,53 @@
-const convertInclude = ({ href, url, type, ...rest }) => {
+import 'isomorphic-fetch';
+import nodeLogger from '#lib/logger.node';
+
+const logger = nodeLogger(__filename);
+
+// this defaults all VJ, IDT2 and IDT1 includes to the specified urls for now
+// should be removed once mozart routes have been created
+const includeUrls = {
+  vj: 'https://simorgh-include-test.s3-eu-west-1.amazonaws.com/vj.html',
+  idt2: 'https://simorgh-include-test.s3-eu-west-1.amazonaws.com/idt2.html',
+  idt1: 'https://simorgh-include-test.s3-eu-west-1.amazonaws.com/idt1.html',
+};
+
+/* This ensures the Include markup is encoded before being added
+   in the page data because serializing window.SIMORGH_DATA with an html string
+    as part of the JSON will result in having some characters unescaped.
+*/
+const encodeHTML = str =>
+  String(str)
+    .replace(/&/g, '&amp;')
+    .replace(/</g, '&lt;')
+    .replace(/>/g, '&gt;')
+    .replace(/"/g, '&quot;')
+    .replace(/'/g, '&#39;');
+
+const fetchMarkup = async url => {
+  try {
+    const res = await fetch(url, { timeout: 3000 });
+    if (res.status !== 200) {
+      throw new Error('Failed to fetch');
+    } else {
+      const html = await res.text();
+      return encodeHTML(html);
+    }
+  } catch (e) {
+    logger.error(
```

Comment:
Can we log the URL we tried to fetch in this event. It would make debugging a lot easier.

Reply:
Also the event should probably be namespaced. Something like `includes_fetch_error` or `smallprox_fetch_error` `http_fetch_error` is very generic.